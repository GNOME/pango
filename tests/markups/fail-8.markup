<b attr="not allowed">test</b>
