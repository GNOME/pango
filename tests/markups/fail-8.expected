ERROR: Tag 'b' does not support attribute 'attr' on line 1 char 31